gatsby/src/pages/faq.js

```diff
@@ -815,7 +815,15 @@ const Faq = ({ data }) => {
                 What clients are available?
               </h4>
               <p>
-                See also:{" "}
+                See also the following lists:{" "}
+                <a href="/clients">
+                  Clients
+                </a>
+                ,{" "}
+                <a href="/clients-matrix">
+                  Clients Matrix
+                </a>
+                , and
```

Comment:
Missing a space here too, and we should be good :)

Reply:
Ugh, you're right! They're not that easy to spot.